src/set_theory/ordinal_arithmetic.lean

```diff
@@ -2098,13 +2096,6 @@ begin
     intro e, simp only [e, mul_zero] }
 end
 
-local infixr ^ := @pow ordinal ordinal ordinal.has_pow
-
-theorem opow_lt_omega {a b : ordinal} (ha : a < omega) (hb : b < omega) : a ^ b < omega :=
```

Comment:
This lemma was removed, is it a trivial consequence of some other lemma?

Reply:
Yes, this is exactly the lemma saying omega is an exponenti princip ordinal.